Cargo.toml

```diff
@@ -9,6 +9,9 @@ version = "0.1.1"
 edition = "2021"
 
 [dependencies]
+anyhow = { version = "1.0" }
```

Comment:
Why not short version (`anyhow = "1.0"`) here instead of braced?

Reply:
Anyway, fixed in 01c7a4650cde0230df514c219733566fa6150807